pkg/app/app_wasm.go

```diff
@@ -139,7 +139,18 @@ func navigate(u *url.URL, updateHistory bool) error {
 
 	root, ok := routes[path]
 	if !ok {
-		root = NotFound
+		for _, r := range routesWithRegexp {
+			// tried MatchString below but it fails one unit test
```

Comment:
Could the test have an error?

Reply:
I don't know. By requiring explicit ^ and $, it's moot.